spec/pluralrules.html

```diff
@@ -327,9 +307,7 @@
     <ul>
       <li>[[Locale]] is a String value with the language tag of the locale whose localization is used by the plural rules.</li>
       <li>[[Type]] is one of the String values *"cardinal"*, or *"ordinal"*, identifying the plural rules used.</li>
-      <li>[[MinimumIntegerDigits]] is a non-negative integer Number value indicating the minimum integer digits to be used. Numbers will be padded with leading zeroes if necessary.</li>
-      <li>[[MinimumFractionDigits]] and [[MaximumFractionDigits]] are non-negative integer Number values indicating the minimum and maximum fraction digits to be used. Numbers will be rounded or padded with trailing zeroes if necessary.</li>
-      <li>[[MinimumSignificantDigits]] and [[MaximumSignificantDigits]] are positive integer Number values indicating the minimum and maximum fraction digits to be shown. Either none or both of these properties are present; if they are, they override minimum and maximum integer and fraction digits – the formatter uses however many integer and fraction digits are required to display the specified number of significant digits.</li>
+      <li>[[NumberFormat]] is an initialized NumberFormat object with locale fixed to *"en"*, used to format Number values to Strings.</li>
```

Comment:
The reason for using the "en" locale is that it effectively makes [FormatNumber](http://www.ecma-international.org/ecma-402/1.0/#FormatNumber) produce output that matches its input exactly. If we use a locale that's user-specific, then e.g. the decimal separator may well be `,` instead of `.`, messing up the stringification for use in GetOperands.

This messiness could be avoided if FormatNumber were split into two abstract operations; one applying the raw stringification (cleaning up the input and calling ToRawPrecision or ToRawFixed), and a second turning that raw string into an ILND representation thereof.

Can we propose changes to FormatNumber, or would that be too much?